app/api/event_copy.py

```diff
@@ -45,13 +46,23 @@ def create_event_copy(identifier):
     discount_codes = DiscountCode.query.filter_by(event_id=event.id).all()
     speaker_calls = SpeakersCall.query.filter_by(event_id=event.id).all()
     user_event_roles = UsersEventsRoles.query.filter_by(event_id=event.id).all()
+    tax = Tax.query.filter_by(event_id=event.id).all()
 
     db.session.expunge(event)  # expunge the object from session
     make_transient(event)
     delattr(event, 'id')
     event.identifier = get_new_event_identifier()
     save_to_db(event)
 
+    # Ensure tax information is copied
+    if len(tax) > 0:
+        tax_object = tax[0]
```

Comment:
@uds5501 Just a question, why tax[0] ?

Reply:
@kushthedude because there is only one tax object per event and `.all()` returns an array.